### ISSUE TITLE
Support status for Django REST framework

### ISSUE THREAD

**Issue by spectras** [KristianOellegaard/django-hvad#224]
This is an informative issue on the current state of [Django REST framework](https://github.com/tomchristie/django-rest-framework).

We are in the process of introducing **official support** for REST framework in hvad. Actually, the process is pretty much complete as we now have full integration support, with thorough testing and documentation. Except this support relies on version 3.1 of REST framework which will be released in March.

This means that for now, if you want to use REST framework with django-hvad, you have 3 options:

- Jump straight to March: use REST framework [from git](https://github.com/tomchristie/django-rest-framework/) together with hvad [`feature/restframework`](https://github.com/spectras/django-hvad/tree/feature/restframework) branch. That branch will be maintained until March, at which point it will be merged into `master`. You can follow #220 to get updates on its status.
- Stick to REST framework 2, and use the glue code from [issue #211](https://github.com/KristianOellegaard/django-hvad/issues/211#issuecomment-62233142). It will be mostly compatible with upcoming official support, though it has much less features.
- Customize the interaction by yourself. Take the glue code from above point as a sample, it is short and straightforward. Open issues here if you need, we'll be glad to help.

Maybe the reasons for waiting until March deserve some explanation. We have long wanted to add out-of-the-box support for REST framework. We knew however that REST framework version 3 would have a lot of changes in the parts hvad would interact with, so we postponed working on full integration until we had a clearer view.

Now REST framework 3 is out, it has many improvements, but it also lost the main API entry points that are required for hvad to integrate correctly. We got in touch with the lead developer of REST framework, who promptly did an amazing work on adding clean extension points. Everything will be wrapped up and included in release 3.1 of REST framework.

**TL;DR:** we had to choose between hacky support right now, or clean, fully-featured support starting from March. We went for the latter, especially since it is easy to get things going with [glue code](https://github.com/KristianOellegaard/django-hvad/issues/211#issuecomment-62233142) in the meantime.

**Comment by spectras** [comment]
**Update: REST framework support has been merged and is available on hvad 1.2.0**.
